dask/tests/test_order.py

```diff
@@ -257,6 +315,7 @@ def test_gh_3055():
 
     dsk = dict(w.__dask_graph__())
     o = order(dsk)
+    assert max(diagnostics(dsk, o=o)[1]) <= 8
```

Comment:
```suggestion
    # Hold at most 8 outputs in memory at any time
    assert max(diagnostics(dsk, o=o)[1]) <= 8
```

Reply:
I didn't add these suggestions. I don't like comments that spell out what the code already says and I think this is mostly due to unfamiliarity with the `diagnostics` function / pressure metric.